pyaedt/generic/farfield_visualization.py

```diff
@@ -0,0 +1,2187 @@
+            font_family="arial",
+            vertical=True,
+            position_x=0.05,
+            position_y=0.65,
+            height=0.3,
+            width=0.06,
+            color=axes_color,
+            title=None,
+            outline=False,
+        )
+
+        cad_mesh = self.__get_geometry()
+
+        data = conversion_function(farfield_data[quantity], function=quantity_format)
+        if not isinstance(data, np.ndarray):  # pragma: no cover
+            self.logger.error("Wrong format quantity")
+            return False
+        max_data = np.max(data)
+        min_data = np.min(data)
+        ff_mesh_inst = p.add_mesh(uf.output, cmap="jet", clim=[min_data, max_data], scalar_bar_args=sargs)
+
+        if cad_mesh:  # pragma: no cover
+
+            def toggle_vis_ff(flag):
+                ff_mesh_inst.SetVisibility(flag)
+
+            def toggle_vis_cad(flag):
+                for i in cad:
+                    i.SetVisibility(flag)
+
+            def scale(value=1):
+                ff_mesh_inst.SetScale(value, value, value)
+                sf = AEDT_UNITS["Length"][self.__model_units]
+                ff_mesh_inst.SetPosition(np.divide(self.origin, sf))
+                ff_mesh_inst.SetOrientation(rotation_euler)
+
+            p.add_checkbox_button_widget(toggle_vis_ff, value=True, size=30)
+            p.add_text("Show Far Fields", position=(70, 25), color=text_color, font_size=10)
+            if not scale_farfield:
+                if self.__is_array:
+                    slider_max = int(np.ceil(np.abs(np.max(self.__array_dimension) / np.min(np.abs(p.bounds)))))
+                else:  # pragma: no cover
+                    slider_max = int(np.ceil((np.max(p.bounds) / 2 / np.min(np.abs(p.bounds)))))
+                slider_min = 0
+            else:
+                slider_max = scale_farfield[1]
+                slider_min = scale_farfield[0]
+            value = slider_max / 3
+
+            p.add_slider_widget(
+                scale,
+                [slider_min, slider_max],
+                title="Scale Plot",
+                value=value,
+                pointa=(0.7, 0.93),
+                pointb=(0.99, 0.93),
+                style="modern",
+                title_height=0.02,
+                color=axes_color,
+            )
+
+            cad = []
+            for cm in cad_mesh:
+                cad.append(p.add_mesh(cm[0], color=cm[1], show_scalar_bar=False, opacity=cm[2]))
+
+            if not show_geometry:
+                p.add_checkbox_button_widget(toggle_vis_cad, value=False, position=(10, 70), size=30)
+                toggle_vis_cad(False)
+            else:
+                p.add_checkbox_button_widget(toggle_vis_cad, value=True, position=(10, 70), size=30)
+
+            p.add_text("Show Geometry", position=(70, 75), color=text_color, font_size=10)
+
+        if image_path:
+            p.show(screenshot=image_path)
+        if show:  # pragma: no cover
+            p.show()
```

Comment:
You should update this to avoid the problem of calling show without specifying autoclose, see https://github.com/ansys/pyaedt/pull/4912 and the issue associated 
Note: I might have missing this in other methods